gpiozero/internal_devices.py

```diff
@@ -483,8 +483,12 @@ def usage(self):
         """
         # XXX Use os.statvfs?
         df = subprocess.Popen(['df', '--output=pcent', self.filesystem], stdout=subprocess.PIPE)
-        output = df.communicate()[0].split(b'\n')[1].split()[0]
-        return float(output.decode('UTF-8').rstrip('%'))
+        try:
```

Comment:
There's no `with` style wrapper for this?

Reply:
There is ... but not for 2.7, annoyingly. However, I've just pushed a better fix for this in another branch (uses statvfs instead of a sub-process). I'll merge that a little later to fix this "properly"